src/main/java/org/primeframework/mvc/util/EncryptionException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.primeframework.mvc.util;
+
+/**
+ * Indicates a problem during cookie encryption/decryption
+ */
+public class EncryptionException extends RuntimeException {
```

Comment:
Please add an author tag for consistency.

Reply:
Removed class, so moot point